VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.spec.ts

```diff
@@ -193,69 +250,85 @@ describe('JudgeContextMenuComponent', () => {
         expect(component.isDroppedDown).toBeTruthy();
     });
 
-    it('should return isClickedOutsideOfOpenMenu true when click event is outside element and menu is open', () => {
-        // Arrange
-        component.isDroppedDown = true;
-        spyOn(elementRef.nativeElement, 'contains').and.returnValue(false);
-        const event = new MouseEvent('click');
-        spyOnProperty(event, 'target').and.returnValue(new EventTarget());
+    describe('isClickedOutsideOfOpenMenu', () => {
+        let event: MouseEvent;
+        let containsSpy: jasmine.Spy;
+        beforeEach(() => {
+            event = new MouseEvent('click');
+            spyOnProperty(event, 'target').and.returnValue(document.createElement('div'));
+            containsSpy = spyOn(component['elementRef'].nativeElement, 'contains');
```

Comment:
I really dislike spying on a private input but I couldn't get the elementRef spy that was injected to work. If anyone has any thoughts or ideas on another way to do this I'd be open to it.